build-config/make/mtdutils.make

```diff
@@ -9,7 +10,8 @@
 # This is a makefile fragment that defines the build of mtdutils
 #
 
-MTDUTILS_VERSION	= 1.5.0
+MTDUTILS_VERSION	?= 1.5.0
+MTDUTILS_LATEST_RELEASE_VERSION = 1.5.2
```

Comment:
This all seems a bit complicated.  Perhaps we just move the version of the mtdutils forward to 1.5.2 for all machines and architectures?  We just have to make sure all the machines continue to compile and build properly.

Moving forward to a newer version of a software package is generally a good thing.